.buildkite/pipelines/pull_request/kbn_handlebars.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - command: .buildkite/scripts/steps/test/kbn_handlebars.sh
+    label: 'Check @kbn/handlebars for upstream differences'
+    agents:
+      queue: ci-group-6
```

Comment:
can we try this on the n2-2-spot or n2-4-spot queues?  not sure on the hardware requirements, mainly noting the spot instance.  group 6 is 6 core 16gb, n2-2 is 2 core 8gb.

Reply:
It worked fine with `n2-2-spot`. Unless you want to try something even smaller, I'll just merge it as is then 🙂